### ISSUE TITLE
Does not work with OmniGraffle

### ISSUE THREAD

**Issue by marlonrichert** [eczarny/spectacle#524]
None of the Spectacle commands work with OmniGraffle 6 on OS X El Capitan.

**Comment by eczarny** [comment]
Out of curiosity, does OmniGraffle work well with Accessibility Inspector?

**Comment by marlonrichert** [comment]
I don't know exactly what Accessibility Inspector is supposed to do, but when I have it open and I move my mouse over different parts of OmniGraffle, Accessibility Inspector shows varying values in its Hierarchy and Attributes fields. You can download a free trial of OmniGraffle to see for yourself: https://www.omnigroup.com/download/

**Comment by eczarny** [comment]
@marlonrichert can you provide more information? Console log messages would be helpful. I don't have OS X El Capitan but OmniGraffle 6.5 seems to work fine with Spectacle on OS X Yosemite.

**Comment by marlonrichert** [comment]
17.2.2016 17.29.34,092 Spectacle[415]: Unable to obtain the accessibility element with the specified attribute: AXFocusedApplication (error code -25212)
17.2.2016 17.29.34,092 Spectacle[415]: Failed to find the application that currently has focus.

**Comment by marlonrichert** [comment]
It seems to fail when there is no component that has focus in the OmniGraffle window. This happens when I do not have anything selected (for example, when I initially open a graffle, or when I deselect the current selection by clicking on the background). As soon as I have anything selected in my diagram, Spectacle starts working again.

**Comment by eczarny** [comment]
Ah, that's expected behavior. The window that you intend to move with Spectacle needs to have focus. Otherwise Spectacle can't find the application that is currently focused (as the error message points out).
